src/catalog/protos/objects.proto

```diff
@@ -664,13 +664,23 @@ message AuditLogEventV1 {
     optional string new_history = 3;
   }
 
+  message ToNewIdV1 {
```

Comment:
I was thinking about this a bit more, what if we re-wrote the entire audit log with the new IDs?

Reply:
Chatted about this during a call and I have a lot of apprehensions about that because it would require we update _all_ data system-wide with the new IDs to retain the data's referential integrity.